src/main/java/jenkins/plugins/http_request/util/HttpClientUtil.java

```diff
@@ -47,15 +47,16 @@ private HttpRequestBase doCreateRequestBase(RequestAction requestAction) throws
         //without entity
     	if (requestAction.getMode() == HttpMode.HEAD) {
 			return new HttpHead(getUrlWithParams(requestAction));
-        } else if (requestAction.getMode() == HttpMode.GET) {
+		} else if (requestAction.getMode() == HttpMode.GET && (requestAction.getRequestBody() == null || requestAction.getRequestBody().isEmpty())) {
```

Comment:
StringUtils.isBlank() would be better so that it would filter out blank strings

Reply:
Using StringUtils.isBlank()  would mean it's not possible to send a body containing only whitechars. Whatever the semantic of such a body could be. Maybe for counting whitespaces and returning the number of chars ;)